components/Globe.tsx

```
@@ -96,7 +96,7 @@ export function Globe({ globeConfig, data }: WorldProps) {
 			_buildData();
 			_buildMaterial();
 		}
-	}, [globeRef.current]);
+	}, [globeRef.current, _buildData, _buildMaterial]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)